### ISSUE TITLE
Briefing

### ISSUE THREAD

**Issue by JohnSquare** [nebrelbug/awesome-for-beginners#4]
I have done a little briefing in a Wiki Zone but i am sure that you have more ideas. For this is this Issue, to do a complete briefing. We can do a Brainstorming, right?

Apart from what has already been written, I would add new functions like
* that the registered users can add new projects to the list
* can see who is working in the different projects
* A rating system for the projects.
* How many developers does a project need info (1/4, 2/5, etc)

**Comment by bgub** [comment]
I think that sounds great! Let's see if we can get started with this. @ascii-dev, @nxv1317, @meikay, and anyone else, what do you think of this?

**Comment by ascii-dev** [comment]
Sure... Sorry I've been off for a while, work stuff, anyways, I'll see what I can come up with in terms of basic design withing a few days and make a PR.

**Comment by Meikay** [comment]
@nebrelbug yeah sounds great! Would you like me to push what I have so far to Github? All I have is the content in html. I didn't design it yet, having input from others would be nice. Maybe we can each work on specific components?

**Comment by ascii-dev** [comment]
@Meikay Kindly push what you have so far to Github so we can all have a look and take it up from there. Thanks

**Comment by ascii-dev** [comment]
@nebrelbug Hi, is there a way to see the current progress and what needs to be done so we can move forward. Thanks.

**Comment by bgub** [comment]
@ascii-dev right now, we have hardly anything. Here are some ideas I have:

1. Switch over to Semantic-UI, since it's more beginner-friendly
2. Write a quick web-scraper that fetches the README of Mungell's repo.
3. Create links to his repo?

**Comment by ascii-dev** [comment]
Ok @nebrelbug, I will work on creating a general structure for the design by this weekend. Also, I have a web-scraper I wrote in Python a while back for a website, I can just refactor it for Mungell's repo. Should be able to have that ready by next weekend.

**Comment by bgub** [comment]
Great, thanks! We might want to write the webscraper in JavaScript, so we don't have to have a backend running, and provide fallback content in case their browser doesn't support it or the internet is down. How does that sound?

**Comment by ascii-dev** [comment]
Sure, sounds good @nebrelbug, haven't tried that before but I guess there's a first time for everything.

**Comment by lloydlions** [comment]
Hi. Is there anything i can help of please.. thanks.

**Comment by lloydlions** [comment]
A suggestion, how about we also setup a proj management something like trello then divide into smaller features/functions. So we can see which what is the status, which are already taken or needs help, and which are still available.

**Comment by bgub** [comment]
@lloydlions that's a great idea! How about instead of Trello, we use Github's Projects. I just created a Project, and I think everyone should be able to add cards? It would be great if you would check it out and see how it looks :).

**Comment by JohnSquare** [comment]
What server will we use? What will we use to share code that we do not use in production?codepen or what? will we use a chat like a slack? I think it is important to specify these things before starting the project. Don't you think so?
